src/identity/mod.rs

```diff
@@ -74,6 +75,33 @@ impl Identity {
         };
         Ok(id)
     }
+
+    /// Return context variables for URL templates.
+    pub fn url_variables(&self) -> HashMap<String, String> {
+        // This explicitly does not include "current_version",
+        // "throttle_permille" and "node_uuid".
+        let mut vars = HashMap::new();
+        vars.insert("basearch".to_string(), self.basearch.clone());
+        vars.insert("group".to_string(), self.group.clone());
+        vars.insert("platform".to_string(), self.platform.clone());
+        vars.insert("stream".to_string(), self.stream.clone());
+        vars
+    }
+
+    /// Return Cincinnati client parameters.
+    pub fn cincinnati_params(&self) -> HashMap<String, String> {
+        let mut vars = HashMap::new();
+        vars.insert("basearch".to_string(), self.basearch.clone());
```

Comment:
I guess we could just get the `HashMap` from `self.url_variables()` and enhance that one instead? Good with keeping it separate too though.

Reply:
I prefer to keep the two split, as they have different purposes and quick inspection is easier this way.